testkit/src/main/scala/org/bitcoins/core/gen/CryptoGenerators.scala

```diff
@@ -4,12 +4,109 @@ import org.bitcoins.core.crypto._
 import org.bitcoins.core.script.crypto.HashType
 import org.bitcoins.core.util.CryptoUtil
 import org.scalacheck.Gen
+import scodec.bits.BitVector
 
 /**
   * Created by chris on 6/22/16.
   */
 sealed abstract class CryptoGenerators {
 
+  object entropy {
```

Comment:
Seems weird to me to have a `object` inside of a class. Any reason we don't just move this into it's own file?

Reply:
Wanted to keep all crypto related generators within one file, and avoid a bunch of name repetition.